dataland-frontend/tests/e2e/fixtures/lksg/LksgPreparedFixtures.ts

```diff
@@ -81,3 +84,45 @@ function manipulateFixtureForLksgDatasetWithLotsOfNulls(fixture: FixtureData<Lks
   fixture.companyInformation.companyName = "lksg-a-lot-of-nulls";
   return fixture;
 }
+
+/**
+ * Generates a Lksg fixture with a dataset with many null values for categories, subcategories and field values
+ * @returns the fixture
+ */
+export function generateOneLksgFixtureWithManyNulls(): FixtureData<LksgData> {
```

Comment:
Why is this method being exported?

Reply:
was copied over from old code and is fixed now